.travis/install.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+set -x
+
+ci_requirements="pip setuptools tox virtualenv"
```

Comment:
I don't think that we need pip, and setuptools here, right?

Reply:
true, let me remove that also.